### ISSUE TITLE
Storybook running in browser and VSCode tab but not in explorer view...

### ISSUE THREAD

**Issue by pkreipke** [orta/vscode-react-native-storybooks#29]
I know the storybook server is working and running: can connect via browser fine - and "Show StoryBook Tab" shows the same browser view inside VSCode.

But what I also have is an empty Storybook section in the Explorer sidebar.

What could I be doing wrong?

**Comment by orta** [comment]
Port?

**Comment by SrujithPoondla** [comment]
+1
Same issue

**Comment by ohtangza** [comment]
I am experiencing the same issue with @pkreipke. Any clue?

**Comment by techieshark** [comment]
Same issue, though this is on a standard react (not React Native) project. 

Was pleased to see Storybook tab shows the localhost storybook website. Should it also be able to show stories in the explorer pane? My stories are in src/stories/SomeComponent.story.js and I have these in vscode user settings:
```
"react-native-storybooks.port": 9009,
"react-native-storybooks.storyRegex": "**src/stories/*.story.*",
```

**Comment by jinshin1013** [comment]
Is this an ongoing discussion? Can't seem to find a solution for this problem.

**Comment by pke** [comment]
Yes this seems to be still an issue. Can't get it to run (with a native Android device).
Custom port is set to 9001 and works with storybook browser.

**Comment by ankitjaiswal1994** [comment]
+1 still not showing on VS Code.
